src/gennylib/CMakeLists.txt

```diff
@@ -101,11 +87,13 @@ endif()
 # Link in OpenSSL
 target_link_libraries(gennylib
     PUBLIC
-        OpenSSL::Crypto OpenSSL::SSL
+        metrics
```

Comment:
Hmmm, I'm surprised that you need this since you link it into `gennylib_core`.

Reply:
good catch - removed